.coveragerc

```diff
@@ -521,6 +521,8 @@ omit =
     homeassistant/components/isy994/switch.py
     homeassistant/components/itach/remote.py
     homeassistant/components/itunes/media_player.py
+    homeassistant/components/jellyfin/__init__.py
+    homeassistant/components/jellyfin/media_source.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.

Reply:
I don't think we should increase the scope after we've started the review unless really needed. We're just prolonging the time to merge then. Better to follow up with improvements in other smaller PRs.

Reply:
Will definitely work on this in a future PR.